scripts/release.sh

```diff
@@ -240,9 +234,6 @@ elif [[ ${TRAVIS_BRANCH} =~ ^release\-[0-9]+\.[0-9]+$ ]]; then
     BRANCH_VERSION=${ver_number[1]}
     # Release with unified branch and file versions
     git_branch_commit_and_release ${FILE_VERSION} ${BRANCH_VERSION} branch rc
-elif [[ $TRAVIS_BRANCH = "develop" ]]; then
```

Comment:
Could you not remove this one? This is the release used in gather to test the last changes :grimacing: